core/src/apps/nem2/sign_tx.py

```diff
@@ -0,0 +1,169 @@
+from trezor.crypto.curve import ed25519
+from trezor.crypto.hashlib import sha3_256
+from trezor.messages.NEM2SignedTx import NEM2SignedTx
+from trezor.messages.NEM2CosignatureSignedTx import NEM2CosignatureSignedTx
+from trezor.messages.NEM2SignTx import NEM2SignTx
+from ubinascii import unhexlify, hexlify
+
+from apps.common import seed
+from apps.common.paths import validate_path
+from apps.nem2 import (
+    CURVE,
+    transfer,
+    mosaic,
+    namespace,
+    metadata,
+    aggregate,
+    hash_lock,
+    secret_lock,
+    multisig,
+    account_restriction
+)
+from apps.nem2.helpers import (
+    validate_nem2_path,
+    NEM2_HASH_ALG,
+    NEM2_TRANSACTION_TYPE_AGGREGATE_COMPLETE,
+    NEM2_TRANSACTION_TYPE_AGGREGATE_BONDED
+)
+from apps.nem2.validators import validate
+
+# Included fields are `size`, `verifiableEntityHeader_Reserved1`,
+# `signature`, `signerPublicKey` and `entityBody_Reserved1`.
+def get_transaction_header_size():
+    return 8 + 64 + 32 + 4
```

Comment:
one of `verifiableEntityHeader_Reserved1` or `entityBody_Reserved1` is not included

Reply:
@evias, it should be:
```
def get_transaction_header_size():
    return 4 + 4 + 64 + 32 + 4
```
The 8 is actually the combination of: size and verifiableEntityHeader_Reserved1. I am breaking it out for clarity.
https://github.com/nemtech/nem2-sdk-typescript-javascript/blob/master/src/model/transaction/Transaction.ts#L363-L367